server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -91,15 +91,11 @@ static void readMappingFile(VolumeManager fs, ImportedTableInfo tableInfo, Strin
 
     Path path = new Path(tableInfo.exportFile);
 
-    BatchWriter mbw = null;
-    ZipInputStream zis = null;
-
-    try {
-      VolumeManager fs = manager.getVolumeManager();
-
-      mbw = manager.getContext().createBatchWriter(MetadataTable.NAME, new BatchWriterConfig());
-
-      zis = new ZipInputStream(fs.open(path));
+    try (
+        BatchWriter mbw =
+            manager.getContext().createBatchWriter(MetadataTable.NAME, new BatchWriterConfig());
+        VolumeManager fs = manager.getVolumeManager();
```

Comment:
[Recent discussion in another ticket](https://github.com/apache/accumulo/pull/2117#discussion_r639859548) makes me think this probably should be moved outside of the try-with-resources block so it is not closed.

Reply:
Possible. How much have you looked into it? I noticed in the test `SortedLogRecoveryTest`, it has the VolumeManager creation in a try with resources block but that is only using the local fs for testing purposes. I do not see anywhere else in the code where that is done, however.

Reply:
Im looking into it now. In this case it looks like it gets the VolumeManager from Manager which is probably needed elsewhere. Also, VolumeManager.close() has a comment saying: 

> close the underlying FileSystems

Which leads  me to believe it will close more than intended.

Reply:
I don't think this should be closed, I think its a shared object in the master that would be used by other threads and task.

Reply:
Yeah, after looking into it more, I agree. I see that the Tserver will call the close function for the filesystem when it gets shut down.

Reply:
Agreed. Made this change in the most recent commit, 3aa1bdf.